src/current/v23.1/timestamp.md

```diff
@@ -66,8 +66,31 @@ CockroachDB supports precision levels from 0 (seconds) to 6 (microseconds) for `
 
 You can use an [`ALTER COLUMN ... SET DATA TYPE`]({% link {{ page.version.version }}/alter-table.md %}#alter-column) statement to change the precision level of a `TIMESTAMP`/`TIMESTAMPTZ`-typed column. If there is already a non-default precision level specified for the column, the precision level can only be changed to an equal or greater precision level. For an example, see [Create a table with a `TIMESTAMP`-typed column, with precision](#create-a-table-with-a-timestamp-typed-column-with-precision).
 
+## Supported casting and conversion
+
+`TIMESTAMP` values can be [cast]({% link {{ page.version.version }}/data-types.md %}#data-type-conversions-and-casts) to any of the following data types:
+
+Type | Details
+-----|--------
+`DECIMAL` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`FLOAT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`TIME` | Converts to the time portion (HH:MM:SS) of the timestamp.
+`INT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
+`DATE` | --
```

Comment:
Are the details intentionally blank? It might be better to spell this out even if it has no effect.

Reply:
Updated to give an example (tested)

> `DATE` | Converts to the date (`YYYY-MM-DD`) of the timestamp and omits the other information.